app/decorators/swagger/project_decorator.rb

```diff
@@ -2,21 +2,29 @@ module Swagger
   class ProjectDecorator < Draper::Decorator
     delegate_all
 
-    def to_swagger
+    def to_swagger(with_api_gateway_integration = false)
```

Comment:
Keyword argument would make the method more explicit here I think `def to_swagger(with_api_gateway_integration: false)`

This way `project.to_swagger(true)` will become `project.to_swagger(with_api_gateway_integration: true)`